scripts/run_integration_tests_aarch64.sh

```diff
@@ -8,9 +8,73 @@ export BUILD_TARGET=${BUILD_TARGET-aarch64-unknown-linux-gnu}
 
 WORKLOADS_DIR="$HOME/workloads"
 WORKLOADS_LOCK="$WORKLOADS_DIR/integration_test.lock"
+EDK2_BUILD_DIR="$WORKLOADS_DIR/edk2_build"
 
 mkdir -p "$WORKLOADS_DIR"
 
+build_edk2() {
+    EDK2_REPO="https://github.com/jongwu/edk2.git"
```

Comment:
Now the EDK2 code is still on review and discussion, not ready on master.

@rbradford Could you help create a branch for this in CH fork of edk2?

Reply:
@michael2012z You should have access now to do that.

Reply:
Hi, @rbradford, sorry I still can't.
I am in group `cloud-hypervisor-write`, but the permission setting for `cloud-hypervisor/edk2` there is `Read`. Could you change it `Write`? Thanks.

Reply:
Can you please try it now @michael2012z ? I've update the group setting to `Write`.

Reply:
Thank you, it works. :)